src/core/subscan.js

```diff
@@ -0,0 +1,28 @@
+const SUBSCAN_URL = 'https://dock.api.subscan.io';
+
+export function fetchTransactions({address, page = 0, row = 50}) {
+  const url = `${SUBSCAN_URL}/api/scan/transfers`;
```

Comment:
we should probably have some input validation on the params

Reply:
fixed